libHVCAENx527App/src/HVCAENx527.c

```diff
@@ -1431,20 +1414,6 @@ CAENx527SetAllChParVal( HVCRATE *cr, char *pname, void *val)
 		}
 #endif	/* CAENHVWrapperVERSION */
 		free( chlist);
-		if( retval != CAENHV_OK)
-		{
-			cr->connected = 0;
-#if CAENHVWrapperVERSION / 100 == 2
-			printf("Lost connection to %s@%s: %s (%d)\n", cr->name, cr->IPaddr, CAENHVGetError(cr->name), retval);
-			CAENHVDeinitSystem( cr->name);
-#else
-			printf("Lost connection to %s@%s: %s (%d)\n", cr->name, cr->IPaddr, CAENHV_GetError(cr->handle), retval);
-			CAENHV_DeinitSystem( cr->handle);
-            cr->handle = -1;
-#endif	/* CAENHVWrapperVERSION */
-		    busyUnlock(cr->crate);
-			return( 3);
-		}
```

Comment:
Should we print an error message? Also would it be worth continuing the loop but noting the failure and eventually returning 3 if at least one of the sets fails?